src/components/Widgets/WidgetContainer.tsx

```diff
@@ -0,0 +1,105 @@
+'use client';
+import { LinkMap } from '@/const/linkMap';
+import { TabsMap } from '@/const/tabsMap';
+import { useWelcomeScreen } from '@/hooks/useWelcomeScreen';
+import { useActiveTabStore } from '@/stores/activeTab';
+import type { StarterVariantType } from '@/types/internal';
+import { ChainId } from '@lifi/sdk';
+import { usePathname } from 'next/navigation';
+import React, { useCallback, useLayoutEffect, useMemo, useState } from 'react';
+import { ThemesMap } from 'src/const/themesMap';
+import { useChainTokenSelectionStore } from 'src/stores/chainTokenSelection';
+import { PartnerThemeFooterImage } from '../PartnerThemeFooterImage';
+import { WidgetContainerBox } from './WidgetsContainer.style';
+
+interface WidgetContainerProps {
+  widgetVariant: StarterVariantType;
+  welcomeScreenClosed: boolean;
+  children: React.ReactNode;
+}
+
+export function WidgetContainer({
+  widgetVariant,
+  welcomeScreenClosed,
+  children,
+}: WidgetContainerProps) {
+  const { activeTab, setActiveTab } = useActiveTabStore();
+  const { setWelcomeScreenClosed } = useWelcomeScreen(welcomeScreenClosed);
+  const pathname = usePathname();
+  const [starterVariantUsed, setStarterVariantUsed] = useState(false);
+  const welcomeScreen = useWelcomeScreen(welcomeScreenClosed);
+
+  const { sourceChainToken, destinationChainToken } =
+    useChainTokenSelectionStore();
+
+  const activeChainAlert =
+    sourceChainToken?.chainId === ChainId.SEI ||
+    destinationChainToken?.chainId === ChainId.SEI ||
+    sourceChainToken?.chainId === ChainId.SOL ||
+    destinationChainToken?.chainId === ChainId.SOL;
```

Comment:
Too much duplication

```suggestion
  const chains = [ChainId.SEI, ChainID.SOL]
  const activeChainAlert = chains.includes(sourceChainToken?.chainId) || chains.includes(destinationChainToken?.chainId)
```

So we can extend it and add new chains to the array